### PR TITLE
React ARIA unions [generated]

### PR DESCRIPTION
**Pull request by turansky**
[Generator](https://github.com/turansky/react-types-kotlin)

cc @sgrishchenko

### CODE REVIEW

File: kotlin-react-dom/src/main/generated/react/dom/AriaChecked.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+package react.dom
+
+@Suppress("NAME_CONTAINS_ILLEGAL_CHARS")
+// language=JavaScript
+@JsName("""({__false__: 'false', mixed: 'mixed', __true__: 'true'})""")
+external enum class AriaChecked {
+    __false__,
+    mixed,
+    __true__,
```
Comment (sgrishchenko) [inline kotlin-react-dom/src/main/generated/react/dom/AriaChecked.kt:11]: If you use higher case naming, you won't have to use these controversial naming for boolean values

Comment (turansky) [reply kotlin-react-dom/src/main/generated/react/dom/AriaChecked.kt:11]: Upper case doesn't look like better solution better solution for me

Comment (sgrishchenko) [reply kotlin-react-dom/src/main/generated/react/dom/AriaChecked.kt:11]: Could you show some examples and I will think what we can do? Or you just want to be aligned with TS/HTML specification?

Comment (turansky) [reply kotlin-react-dom/src/main/generated/react/dom/AriaChecked.kt:11]: Yes, I want to be aligned with TS
* to simplify search
* to simplify next migration (when unions will be supported in Kotlin)

**Comment by Leonya** [comment]
Released in pre.232

**Pull request merged**